### PR TITLE
Progress bar

### PR DESCRIPTION
**Pull request by Quaquaro**

### CODE REVIEW

File: src/components/SteppedProgress.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+SteppedProgress.defaultProps = {
+  step: 1
+};
+
+export default function SteppedProgress({ step }) {
+  const getColorNeonBlue = ({ theme }) => theme.color.neonBlue;
+
+  return (
+    <Container>
+      <Progress></Progress>
```
Comment (codingoak) [inline src/components/SteppedProgress.js:18]: You can replace it with a self-closing tag: <Progress />

File: src/components/SteppedProgress.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+          <Step bgcolor={getColorNeonBlue} />
+          <Step />
+        </StepsContainer>
+      )}
+      {step === 2 && (
+        <StepsContainer>
+          <Step bgcolor={getColorNeonBlue} />
+          <Step bgcolor={getColorNeonBlue} />
+        </StepsContainer>
+      )}
```
Comment (codingoak) [inline src/components/SteppedProgress.js:30]: Maybe you can refactor this code later?

**Review by codingoak — APPROVED** [review]
Great work!

File: src/components/SteppedProgress.js (hunk 1 above)
Comment (Quaquaro) [reply src/components/SteppedProgress.js:18]: Thank you for the hint!

File: src/components/SteppedProgress.js (hunk 2 above)
Comment (Quaquaro) [reply src/components/SteppedProgress.js:30]: I hope so...

**Pull request merged**